app/controllers/api/v0/swagger_controller.rb

```diff
@@ -37,13 +37,20 @@ class Api::V0::SwaggerController < ApplicationController
     key :produces, ['application/json']
   end
 
-  SWAGGERED_CLASSES = [
-    Api::V0::SwaggerResponses,
-    Api::V0::EventsSwagger,
-    self
-  ].freeze
+  def self.swagger_classes
+    list = [
+      Api::V0::SwaggerResponses,
+      Api::V0::EventsSwagger,
+      self
+    ]
+    # dyamically add the (avro) datum classes to the swagger. this will result
+    # in the ability for the client to use the generated swagger to build the
+    # correct payload for the event data. Another result is that there are two
+    # levels of validation: incoming in swagger, and thru avro schema validation.
+    list.concat(DatumSwaggerFinder.new.datum_classes)
```

Comment:
still `datum` doesn't do a lot for me.

`SwaggerEventSchemas.all_classes` or something.  Maybe "datum" means "event" here?